source/adios2/core/Engine.inl

```diff
@@ -0,0 +1,129 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ *
+ * Engine.inl
+ *
+ *  Created on: May 17, 2017
+ *      Author: William F Godoy godoywf@ornl.gov
+ */
+
+#ifndef ADIOS2_CORE_ENGINE_INL_
+#define ADIOS2_CORE_ENGINE_INL_
+#ifndef ADIOS2_CORE_ENGINE_H_
+#error "Inline file should only be included from it's header, never on it's own"
+#endif
+
+#include "Engine.h"
```

Comment:
Remove include.  The previous `ifdef` check guarantees this is always being included by `Engine.h`